src/benchmark/PersistenceBenchmark/Messages.cs

```diff
@@ -1,6 +1,6 @@
 ﻿//-----------------------------------------------------------------------
 // <copyright file="Messages.cs" company="Akka.NET Project">
-//     Copyright (C) 2009-2016 Typesafe Inc. <http://www.typesafe.com>
+//     Copyright (C) 2009-2016 Lightbend Inc. <http://www.Lightbend.com>
```

Comment:
Not that it makes a to much of a difference, but shouldn't that be http://www.lightbend.com